src/widgets/Step.js

```diff
@@ -0,0 +1,158 @@
+/* eslint-disable react/forbid-prop-types */
```

Comment:
Do we need the `forbid-prop-types`?

Reply:
yeah - I still need to get round to putting it in the eslint config (object)